Source/Core/InputCommon/DynamicInputTextureManager.h

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/IniFile.h"
+#include "InputCommon/DynamicInputTextureConfiguration.h"
```

Comment:
This can be replaced with a forward declaration if the constructor and destructor of `DynamicInputTextureManager` are defaulted into the cpp file.